processor/processorhelper/metadata.yaml

```diff
@@ -30,7 +30,7 @@ telemetry:
         value_type: int
         monotonic: true
 
-    processor_accepted_spans:
+    processor_memory_limiter_accepted_spans:
```

Comment:
These metrics will need to be moved to the memorylimiter's metadata configuration https://github.com/open-telemetry/opentelemetry-collector/blob/main/processor/memorylimiterprocessor/metadata.yaml

With the existing metrics staying deprecated for another few versions in the processorhelper

Reply:
Ah yep makes sense. I'm going to put this back to draft to make updates